config.yaml

```diff
@@ -2,6 +2,10 @@
 # See LICENSE file for licensing details.
 
 options:
+  enablePasswordDB:
```

Comment:
please follow style of the name convention :)

Reply:
Well-spotted. The files were updated following the naming convention. Thanks